sdk/metricsadvisor/azure-ai-metricsadvisor/tests/conftest.py

```diff
@@ -3,12 +3,3 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # -------------------------------------------------------------------------
-
```

Comment:
You can even remove the file

Reply:
It's going to get added back in the test proxy PR so I decided to keep it 😄